sdk/search/search-documents/api-extractor.json

```diff
@@ -27,5 +27,6 @@
         "logLevel": "none"
       }
     }
-  }
+  },
+  "newlineKind": "lf"
```

Comment:
Unless all other api-extractor.json include this field I'd recommend removing it just to be consistent - I suspect (read: wild guess 😄 ) folks' git config already does the right thing but maybe you found an instance where it did not?

Reply:
I've been experimenting with different tooling, and some of it doesn't respect git's `core.autocrlf` flag. Maybe there's something I can do locally since this problem is virtually isolated to me.